bin/prod-init.sh

```diff
@@ -14,10 +14,11 @@ cp -v $VALUES_DIR/wire-server/{prod-values.example,values}.yaml
 cp -v $VALUES_DIR/wire-server/{prod-secrets.example,secrets}.yaml
 cp -v $VALUES_DIR/databases-ephemeral/{prod-values.example,values}.yaml
 cp -v $VALUES_DIR/fake-aws/{prod-values.example,values}.yaml
-cp -v $VALUES_DIR/nginx-ingress-controller/{prod-values.example,values}.yaml
+cp -v $VALUES_DIR/ingress-nginx-controller/{prod-values.example,values}.yaml
 cp -v $VALUES_DIR/nginx-ingress-services/{prod-values.example,values}.yaml
 cp -v $VALUES_DIR/nginx-ingress-services/{prod-secrets.example,secrets}.yaml
 cp -v $VALUES_DIR/demo-smtp/{prod-values.example,values}.yaml
+cp -v $VALUES_DIR/ingress-nginx-controller/{prod-values.example,values}.yaml
```

Comment:
Duplicating line 17. Is this needed? That it is a `cp` call makes me think that it isn't.

Reply:
thanks for pointing out..fixed